invenio_records_marc21/api.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
+#
+# invenio-records-marc21 is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Marc21 Api."""
+
+from __future__ import absolute_import, print_function
+
+from invenio_drafts_resources.records import Draft, Record
+from invenio_records.systemfields import RelationsField
+from invenio_records_resources.records.systemfields import IndexField
+from werkzeug.local import LocalProxy
+
+from . import models
+
+
+class Marc21Draft(Draft):
+    """Marc21 draft API."""
+
+    model_cls = models.DraftMetadata
+
+    index = IndexField("marc21-drafts-marc21-v1.0.0", search_alias="marc21-drafts")
```

Comment:
does marc21 need a draft record creation? @utnapischtim

Reply:
It sounds to me like a good idea to make a draft first. In the feature, the module is used to import vom campus online and send it further to alma where the record will be modified again and then it'll be published.